versions/3.0.md

```diff
@@ -1998,293 +1915,136 @@ paths:
-          description: a pull request object
-          content:
-            application/json: 
-              schema: 
-                $ref: '#/components/schemas/pullrequest'
-          links:
-            pullRequestMerge:
-              $ref: '#/components/links/PullRequestMerge'
-  /2.0/repositories/{username}/{slug}/pullrequests/{pid}/merge: 
-    post: 
-      operationId: mergePullRequest
-      parameters: 
-      - name: username
-        in: path
-        required: true
-        schema:
-          type: string
-      - name: slug
-        in: path
-        required: true
-        schema:
-          type: string
-      - name: pid
-        in: path
-        required: true
-        schema:
-          type: string
-      responses: 
-        '204':
-          description: the PR was successfully merged
-components:
-  links:
-    UserRepositories:
-      # returns array of '#/components/schemas/repository'
-      operationId: getRepositoriesByOwner
-      parameters:
-        username: $response.body#/username
-    UserRepository:
-      # returns '#/components/schemas/repository'
-      operationId: getRepository
-      parameters:
-        username: $response.body#/owner/username
-        slug: $response.body#/slug
-    RepositoryPullRequests:
-      # returns '#/components/schemas/pullrequest'
-      operationId: getPullRequestsByRepository
-      parameters: 
-          username: $response.body#/owner/username
-          slug: $response.body#/slug
-    PullRequestMerge:
-      # executes /2.0/repositories/{username}/{slug}/pullrequests/{pid}/merge
-      operationId: mergePullRequest
-      parameters:
-        username: $response.body#/author/username
-        slug: $response.body#/repository/slug
-        pid: $response.body#/id
-  schemas: 
-    user: 
-      type: object
-      properties: 
-        username: 
-          type: string
-        uuid: 
-          type: string
-    repository: 
-      type: object
-      properties: 
-        slug: 
-          type: string
-        owner: 
-          $ref: '#/components/schemas/user'
-    pullrequest: 
-      type: object
-      properties: 
-        id: 
-          type: integer
-        title: 
-          type: string
-        repository: 
-          $ref: '#/components/schemas/repository'
-        author: 
-          $ref: '#/components/schemas/user'
-```
+##### OperationRef Examples
 
 As references to `operationId` MAY NOT be possible (the `operationId` is an optional 
 value), references MAY also be made through a relative `operationRef`:
 
 ```yaml
-components:
-  links:
-    UserRepositories:
-      # returns array of '#/components/schemas/repository'
-      operationRef: '#paths~12.0~1repositories~1{$response.body#/username}'
+links:
+  UserRepositories:
+    # returns array of '#/components/schemas/repository'
+    operationRef: '#paths~12.0~1repositories~1/{username}'
+    parameters:
+      username: $response.body#/username
 ```
 
 or an absolute `operationRef`:
```

Comment:
Link back to `#linkOperationId`, or if you add a section for **Operation Reference**, link to that.